critiquebrainz/data/model/user.py

```diff
@@ -97,7 +97,7 @@ def avatar(self):
         if self.show_gravatar and self.email:
             return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=identicon&r=pg"
         else:
-            return "https://gravatar.com/avatar/" + hashlib.md5(self.id.replace("-", "").encode("ascii")).hexdigest() + "?d=identicon"
+            return "https://gravatar.com/avatar/" + hashlib.md5(self.id.encode("ascii")).hexdigest() + "?d=identicon"
```

Comment:
No, it was ascii earlier, so I thought it was best to keep it that way.
I'll change this.